tests/test_biolink_model/test_biolink_model.py

```diff
@@ -57,7 +57,11 @@
 )
 def test_biolink(generator, extension, gen_kwargs, serialize_kwargs, temp_dir, snapshot, input_path):
     BIOLINK_YAML = input_path("biolink-model.yaml")
-    if not extension.startswith("."):
+    if extension == ".graphql":
+        with pytest.raises(NotImplementedError):
+            generated = generator(BIOLINK_YAML, **gen_kwargs).serialize(**serialize_kwargs)
+        print("jarl")
```

Comment:
who is `jarl` lol

Reply:
Also a debugging output that slipped out on late-night commits 😇

`jarl` is just a word that is almost non-existent in code and I use for easy searching in big text blocks.

Also removing.

Cultural note: it is also an exclamation that a [Spanish comedian of my youth](https://en.wikipedia.org/wiki/Chiquito_de_la_Calzada) used to say. Another exclamation that he used and I also use to simplify searches on debugging is `comorl` 😅

Reply:
i love this